### PR TITLE
Update castBar.lua

### PR DESCRIPTION
**Pull request by Goranaws**
Complete Rewrite.

### CODE REVIEW

**Comment by Goranaws** [comment]
I really like what I've come up with. I think you will be pleased.

File: castBar.lua [hunk 1]
```diff
@@ -1,335 +1,397 @@
...
+	return target
 end
 
--- Core Code
-function CastBar:New()
-	local frame = CastBar.proto.New(self, 'cast')
+function Frame:Create(...)
+	local bar = Frame.proto.Create(self, ...)
+	
+	bar.cast = CreateFrame("StatusBar",  bar:GetName().."Bar", bar, "DominosCastingBarFrameTemplate")
```
Comment (Tuller) [inline castBar.lua]: DominosCastingBarFrameTemplate - Should that be CastingBarFrameTemplate?

Comment (Goranaws) [inline castBar.lua]: Yes. I'm still working some things out. You can expect another update by
this weekend.

**Comment by Goranaws** [comment]
Updated. Made major changes to the texture selector, enjoy. 

also, I need to fix cast text and cast time overlap.

**Pull request merged**